cpp/Platform.Interfaces.Tests/Platform.Interfaces.Tests.cpp

```diff
@@ -3,162 +3,162 @@
 
 namespace Platform::Interfaces::Tests
 {
-    TEST(CompileTests, ICounter)
+    TEST(CompileTests, Counter)
     {
-        struct EmptyCounter
+        struct EmptyCounter: ICounter<int>
         {
             int Count() { return {}; }
         };
         // You can use
-        static_assert(ICounter<EmptyCounter, int>);
+        static_assert(CCounter<EmptyCounter, int>);
         // to avoid mismatching the interface at compile time
 
-        struct EmptyCounterInt
+        struct EmptyCounterInt: public ICounter<int, int>
```

Comment:
You can name this struct as `EmptyCounter2`

Reply:
Ok